internal/elasticsearch/common.go

```diff
@@ -637,14 +637,49 @@ func newVolumeSource(clusterName, nodeName, namespace string, node api.Elasticse
 		StorageClassName: specVol.StorageClassName,
 	}
 
-	err := persistentvolume.CreateOrUpdatePVC(context.TODO(), client, pvc, persistentvolume.LabelsEqual, persistentvolume.MutateLabelsOnly)
+	// PersistentVolumeClaims can run into ResourceQuota error. Since the create
```

Comment:
Why are we hitting the API server in what amounts to a contructor (e.g. newXXXXX).  The original method should have never been their either